framework/common/theme/Color.kt

```diff
@@ -66,9 +68,12 @@ object Color {
             val relation: Color,
             val attribute: Color,
         )
+        override fun toString(): String {
+            return this.name
+        }
```

Comment:
`Dropdown`s call `toString()` on their values to provide strings for users to select. We want sane ones for the preference dialog.